src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -83,6 +90,10 @@ private static long getThreadId(ExecutorService executor)
     private final ProgressLog progressLog;
     private final RangesForEpochHolder rangesForEpochHolder;
 
+    // TODO (expected): schedule regular pruning of this collection, and this is not persistent!
+    @Nullable
+    ReducingRangeMap<Timestamp> rejectBefore;
```

Comment:
should this be addressed? I think persistence could be mostly copied from paxos repair history as well, and scheduling a periodic command store maintenance task is pretty reasonable.

Reply:
This is from Benedict's exclusive sync code. He might have something newer for this?

It isn't used in live migration because we don't do exclusive sync for live migration.